### PR TITLE
Add additional assets for aws provider

### PR DESCRIPTION
**Pull request by RamanaReddy0M**
closes:
- #508

### CODE REVIEW

**Comment by RamanaReddy0M** [comment]
```console

✗ ./cloudlist -p aws -s ecs,cloudfront

  _______             _____     __ 
 / ___/ /__  __ _____/ / (_)__ / /_
/ /__/ / _ \/ // / _  / / (_-</ __/
\___/_/\___/\_,_/\_,_/_/_/___/\__/ 

                projectdiscovery.io

[INF] Current cloudlist version 1.0.8 (latest)
[INF] Listing assets from provider: aws services: cloudfront,ecs id: staging
plobuild.com
75.2.70.75
_6c769667fbb558e7887da1e303d13531.plobuild.com
_478fdab93574403bba4208cd4581a008.acm-validations.aws.
...
```

File: pkg/providers/aws/eks.go [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+		if aws.StringValue(clustersOutput.NextToken) == "" {
+			break
+		}
+		req.SetNextToken(*clustersOutput.NextToken)
+	}
+	return nil
+}
+
+func newClientset(cluster *eks.Cluster) (*kubernetes.Clientset, error) {
+	// fmt.Printf("%+v", cluster)
```
Comment (ShubhamRasal) [inline pkg/providers/aws/eks.go:111]: remove debug line

**Review by ShubhamRasal — CHANGES_REQUESTED** [review]

File: pkg/providers/aws/eks.go [hunk 2]
```diff
@@ -0,0 +1,140 @@
...
+		regionName := *region.RegionName
+		ep.eksClient = eks.New(ep.session, aws.NewConfig().WithRegion(regionName))
+		_ = listEKSResources(ep.eksClient, list)
+	}
+	return list, nil
+}
+
+func listEKSResources(eksClient *eks.EKS, list *schema.Resources) error {
+	req := &eks.ListClustersInput{
+		MaxResults: aws.Int64(100),
```
[the remark this answers is not in the document]
Comment (RamanaReddy0M) [reply pkg/providers/aws/eks.go]: refactored to listEKSResources(eksClient *eks.EKS) (*schema.Resources, error).

File: internal/runner/options.go [hunk 3]
```diff
@@ -65,6 +66,7 @@ func ParseOptions() *Options {
 		flagSet.StringSliceVar(&options.Id, "id", nil, "display results for given ids (comma-separated)", goflags.NormalizedStringSliceOptions),
 		flagSet.BoolVar(&options.Hosts, "host", false, "display only hostnames in results"),
 		flagSet.BoolVar(&options.IPAddress, "ip", false, "display only ips in results"),
+		flagSet.StringSliceVarP(&options.Services, "service", "s", nil, "query and display results from given service (comma-separated))", goflags.NormalizedStringSliceOptions),
```
Comment (dogancanbakir) [inline internal/runner/options.go:69]: I believe, we can use enumslicevarp instead https://github.com/projectdiscovery/goflags/blob/main/examples/basic/main.go#L45

File: pkg/providers/aws/alb.go [hunk 4]
```diff
@@ -0,0 +1,104 @@
...
+	}
+	return list, nil
+}
+
+func listELBV2Resources(albClient *elbv2.ELBV2, ec2Client *ec2.EC2) (*schema.Resources, error) {
+	list := schema.NewResources()
+	req := &elbv2.DescribeLoadBalancersInput{
+		PageSize: aws.Int64(20),
+	}
+	for {
```
Comment (dogancanbakir) [inline pkg/providers/aws/alb.go:42]: How about dividing them into funcs to make them more readable?

**Review by dogancanbakir — APPROVED** [review]

File: internal/runner/options.go (hunk 3 above)
Comment (ehsandeep) [reply internal/runner/options.go:69]: `enumslicevarp` can be used for provider option as well. cc @RamanaReddy0M

**Review by ehsandeep — CHANGES_REQUESTED** [review]

**Review by ehsandeep — CHANGES_REQUESTED** [review]
merge conflict

**Comment by RamanaReddy0M** [comment]
```console
✗ ./cloudlist -p gcp,aws -s s3

  _______             _____     __ 
 / ___/ /__  __ _____/ / (_)__ / /_
/ /__/ / _ \/ // / _  / / (_-</ __/
\___/_/\___/\_,_/\_,_/_/_/___/\__/ 

                projectdiscovery.io

[INF] Current cloudlist version 1.0.8 (latest)
[INF] Listing assets from provider: aws services: s3 id: dev
www.itlumee.in.s3.amazonaws.com
[INF] Found 1 Hosts for aws (dev)
[INF] Listing assets from provider: gcp services: s3 id: dev
gcf-sources-4200296701-us-central1.storage.googleapis.com
pd-cloudlist-test-bucket.storage.googleapis.com
[INF] Found 2 Hosts for gcp (dev)
```

**Review by ehsandeep — APPROVED** [review]

**Pull request merged**